src/kv/RocksDBStore.cc

```diff
@@ -1248,7 +1248,7 @@ int RocksDBStore::do_open(ostream &out,
   ceph_assert(default_cf != nullptr);
   
   PerfCountersBuilder plb(cct, "rocksdb", l_rocksdb_first, l_rocksdb_last);
-  plb.add_time_avg(l_rocksdb_get_latency, "get_latency", "Get latency");
+  plb.add_time_avg(l_rocksdb_get_latency, "get_latency", "Get latency", NULL, PerfCountersBuilder::PRIO_USEFUL);
```

Comment:
I know most of the places (like BlueFS) use `NULL` in PerfCounterBuilder. But lets replace with `nullptr`.

Reply:
Sure, thanks so much for the review. :) I have fixed this and pushed again.